corehq/apps/app_manager/templates/app_manager/partials/releases/releases_deploy_modal.html

```diff
@@ -91,35 +91,37 @@ <h4 class="panel-title">
-                                                        openRemoteModal: full_odk_install_url,
-                                                        click: clickScan
-                                                    ">
-                            {% trans "Scan Application Barcode" %}
-                          </a>
-                        </p>
-                        <p>
-                          <a href="#" data-bind="
-                                                        click: click_app_code,
-                                                        visible: !app_code() && !failed_url_generation() && !generating_url()
-                                                        ">
-                            {% trans "Enter App Code" %}
-                          </a>
-                          <span data-bind="visible: app_code">
-                                                        {% trans "Enter app code on installation screen: " %}
-                                                        <code class="bitly" data-bind="text: app_code"></code>
-                                                    </span>
-                          <i data-bind="visible: generating_url()" class='fa fa-spin fa-spinner'></i>
-                          <span class="text-warning" data-bind="
-                                                        visible: failed_url_generation() && !generating_url()
-                                                        ">
-                                                        {% blocktrans %}
-                                                          (No app code available. If you would like to use an app code instead
-                                                          of scanning the application barcode, try making another version.)
-                                                        {% endblocktrans %}
-                                                    </span>
-                        </p>
+                        <ul>
+                          <li>
+                            <a href="#" data-bind="
+                                                          openRemoteModal: full_odk_install_url,
+                                                          click: clickScan
+                                                      ">
+                              {% trans "Scan Application Barcode" %}
+                            </a>
+                          </li>
+                          <li>
+                            <a href="#" data-bind="
+                                                          click: click_app_code,
+                                                          visible: !app_code() && !failed_url_generation() && !generating_url()
+                                                          ">
+                              {% trans "Enter App Code" %}
+                            </a>
+                            <span data-bind="visible: app_code">
```

Comment:
i always opt for hidden elements instead of `<span>`
```suggestion
                            <!-- ko visible: app_code -->
```

Reply:
that is in the case the span is not doing anything meaningful stylistically to the UI

Reply:
Yeah, I see what you mean. In this case I didn't actually introduce or edit this code, I just indented it. Here's that same diff without whitespace. https://github.com/dimagi/commcare-hq/pull/33346/files?diff=unified&w=1.

In the interest of getting this out quickly enough to avoid hitting our new link limit, I'd like not to get into improving the code style beyond what I need to get the minimal safe changes in. Would you be alright with that?

Reply:
yes, that sounds good!